views/includes/check-your-answers/beneficial-owner-gov.html

```diff
@@ -31,38 +31,44 @@
   {% include "includes/check-your-answers/nocs/beneficial-owner-noc.html" %}
 {% endset %}
 
-{% if not FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC %}
-  {% set bogBeneficialOwnerNonLegalFirmNOCs %}
-    {% set boNOCNonLegalTypesArray = bog.non_legal_firm_members_nature_of_control_types %}
-    {% include "includes/check-your-answers/nocs/beneficial-owner-member-of-firm-noc.html" %}
-  {% endset %}
-{% else %}
-  {% set bogBeneficialOwnerTrustControlNOCs %}
-    {% set boNOCTrustControlTypesArray = bog.trust_control_nature_of_control_types %}
-    {% include "includes/check-your-answers/nocs/beneficial-owner-trust-control-noc.html" %}
-  {% endset %}
-
-  {% set bogBeneficialOwnerNonLegalFirmControlNOCs %}
-    {% set boNOCNonLegalFirmControlTypesArray = bog.non_legal_firm_control_nature_of_control_types %}
-    {% include "includes/check-your-answers/nocs/beneficial-owner-member-of-firm-control-noc.html" %}
-  {% endset %}
-
-  {% set bogOwnerOfLandPersonNOCs %}
-    {% set boNOCOwnerOfLandPersonJurisdictionsArray = bog.owner_of_land_person_nature_of_control_jurisdictions %}
-    {% include "includes/check-your-answers/nocs/beneficial-owner-owner-of-land-person-noc.html" %}
-  {% endset %}
-
-  {% set bogOwnerOfLandOtherEntityNOCs %}
-    {% set boNOCOwnerOfLandOtherEntityJurisdictionsArray = bog.owner_of_land_other_entity_nature_of_control_jurisdictions %}
-    {% include "includes/check-your-answers/nocs/beneficial-owner-owner-of-land-other-entity-noc.html" %}
-  {% endset %}
-{% endif %}
+{% set bogBeneficialOwnerNonLegalFirmNOCs %}
```

Comment:
Does this mean we'll be showing the new nocs even when the feature flag is false?

Reply:
no, the new Nocs will be hidden when flag is false. These sections are just to build the html if a noc is present. For the no change journey, the old invalid noc is to be displayed if present in CHS so they could potentially have all 7 displayed for a no change journey. The page will construct the html depending on what nocs they tick so we can just let them do their job and then control what is displayed further down when we join the noc html together to display using the boNOCHtml variable.

Reply:
Yes I see that now.